app/models/hackbot/helpers.rb

```diff
@@ -18,6 +18,12 @@ def action
       event[:action]
     end
 
+    # Returns whether the current event being processed is a PM. Slack channel
+    # IDs start with D if they're a private message.
+    def in_pm?
```

Comment:
On second thought, I'll merge for the time being and make a PR to remove the extra code right after.